test/models/tag_test.rb

```diff
@@ -0,0 +1,8 @@
+require 'test_helper'
+class TagTest < ActiveSupport::TestCase
+  test "it returns users following this tag but not given tags" do
+    tag_followers = [1,4,5]
```

Comment:
Looking cool, but where do these numbers come from? I think we should try to show, in the test, that these users are following the tag -- assert that in a sequence of assertions here to help demonstrate the test more clearly. 

Also, rather than using the uids directly, best use `ruser(:Bob)` and so forth -- use them via the standard syntax for fixtures. Make sense?